File: quizii/app/src/main/java/com/example/quizii/MainActivity.java
package com.example.quizii;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    public void install (View view)
    {
        Button instali = (Button) findViewById(R.id.install);
        Intent instalaciaintent = new Intent(this, instalacia.class);
        startActivity(instalaciaintent);
    }

    public void problemebi (View view)
    {
        Intent fixebiii = new Intent(this, fixebi.class);
        startActivity(fixebiii);
    }

    public void garemo (View view)
    {
        Intent garemo = new Intent(this, garemosmowyoba.class);
        startActivity(garemo);
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: java_and_kotlin_g11/app/src/main/java/com/example/java_and_kotlin_g11/MainActivity.kt
package com.example.java_and_kotlin_g11

import android.os.Bundle
import android.view.View
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import java.lang.Math.random
import java.util.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val myperson = person( "Kotlin" , 23)
        val textview = findViewById<TextView>(R.id.textview)

        textview.text = greet(myperson.getName())


        val value = generatenumber()
        val numberView = findViewById<TextView>(R.id.numberview)
        numberView.text = (value.toString())


    }
    fun updatenumber (view: View){
        val value = generatenumber()
        val numberView = findViewById<TextView>(R.id.numberview)
        numberView.text = (value.toString())
    }
}


fun generatenumber(): Int{
    val rand = Random()
    val value: Int = rand.nextInt(50)
    return value
}


fun greet(text: String = ""): String{
   return "hi ${text}"
}